packages/marketplace/src/components/ReadmePage.tsx

```diff
@@ -70,9 +70,12 @@ class ReadmePage extends React.Component<
     // Make the page scrollable
     const main = document.querySelector("#marketplace-readme")?.closest("main");
     if (main) {
-      setTimeout(() => {
+      const callScrollbar = setInterval(() => {
         // TODO: see if it's possible to use some load event or mutation observer to do this
+        main.style.overflowY = "visible";
         main.style.overflowY = "auto";
```

Comment:
Why is it setting this and then resetting it?

Reply:
So that it works in the way that it is *re-applying* the stylings. Setting it to the same value the second time doesn't change the behavior.

Reply:
This is so messed up 😂